muspinsim/muspinsim.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="muspinsim" name="Simulate muon experiment spin dynamics (muspinsim)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip install git+https://github.com/muon-spectroscopy-computational-project/muspinsim.git &&
+        ln -s $mu_sim $mu_sim.name &&
+        ln -s $mu_exp_in $mu_exp_in.name &&
+        ## grab 'name' field from input file, if it doesn't exist, set name as 'muspinsim'
+        mu_out=\$(grep -A1 '^name' $mu_sim.name | grep -v 'name' | sed -e 's/^[ \t]*//') &&
+        mu_out=\${mu_out:-"muspinsim"}.dat &&
+        log_out=\$(echo $mu_sim.name | cut -d '.' -f1).log &&
+        ## if fitting data required, change filepath in input file to point to symlink
+        if grep '^fitting_data' $mu_sim; then sed -iE "/^fitting_data/{n;s/([^)]*)/(\"$mu_exp_in.name\")/g}" $mu_sim.name; fi &&
```

Comment:
If `fitting_data` is present in the input, but `$mu_exp_in` is not set, this causes a somewhat cryptic error `OSError: None not found.` We should check that `$mu_exp_in` is set in this case and break with a more user-friendly error message if it's not.